src/dependencies.stanza

```diff
@@ -32,20 +38,26 @@ defn resolve-deps (cfg:SlmToml) -> Tuple<Dependency> :
   ;    using only the lockfile.
   ;
   ; 2. Otherwise we use the slm.toml
-  if file-exists?(SLM_LOCK_NAME):
-    val lock-cfg = within f = open(SLM_LOCK_NAME):
-      parse-slm-lock-file(f)
-    parse-slm-lock-and-resolve-dependencies(cfg, lock-cfg)
-  else:
+  ;JITX-8200;if file-exists?(SLM_LOCK_NAME):
+  ;JITX-8200;  val lock-cfg = within f = open(SLM_LOCK_NAME):
+  ;JITX-8200;    debug("resolve-deps: calling parse-slm-lock-file")
+  ;JITX-8200;    parse-slm-lock-file(f)
+  ;JITX-8200;  debug("resolve-deps: calling parse-slm-lock-and-resolve-dependencies")
+  ;JITX-8200;  parse-slm-lock-and-resolve-dependencies(cfg, lock-cfg)
+  ;JITX-8200;else:
```

Comment:
I propose that we just remove these commented sections here and elsewhere. The `git` history does a better job of tracking this implementation.

When we get to a point where we want to re-introduce the lock file, I think enough things will have changed that this won't be very valuable anymore. 

Forward unto glory :)